setup.py

```diff
@@ -94,7 +91,11 @@ def main():
                       (data_dir, accounting_files),
                       (data_dir, message_files),
                       (data_dir, update_scripts),
-                      (log_rotate_dir, log_rotate_files)],
+                      (log_rotate_dir, log_rotate_files),
+                      # Create empty directories
+                      ('/var/log/apel', []),
+                      ('/var/run/apel', [])
```

Comment:
You should be able to add a trailing `,` at the end of this line, Python will ignore and you will be able to cleanly add more entries later.

Reply:
Oh, yes. I forgot to add this in my suggestion.

Reply:
done